rfcs/20190610-standartizing-composite_ops.md

```diff
@@ -0,0 +1,491 @@
+# Standardizing composite ops in tensorflow.
```

Comment:
```suggestion
# Standardizing Composite Ops for Inference in TensorFlow.
```

Reply:
I think this is not entirely accurate, since we expect this standartization to eventually extend to gradient as well. I've added several statements in the first three paragraph to that extent.

Reply:
This still seems to describe the current proposal as it stands: this RFC by itself only enables inference tooling. It seems to me that  "we leave the door open for follow-up gradient
 optimization support" acknowledge this.